capstone/src/main/java/com/google/sps/servlets/ClubServlet.java

```diff
@@ -20,4 +31,42 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.setContentType("text/html;");
     response.getWriter().println(json);
   }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    UserService userService = UserServiceFactory.getUserService();
+    String founderEmail = userService.getCurrentUser().getEmail();
+
+    String clubName = request.getParameter("name");
+    String description = request.getParameter("description");
+    String website = request.getParameter("website");
+    BlobKey key = getBlobKey(request, "logo");
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+
+    Entity clubEntity = new Entity("Club", clubName);
```

Comment:
So for now, we don't have a check for duplicate names, right? Or does datastore take care of that automatically?

Reply:
I added a confirmation/error page that the servlet redirects to so if the name is duplicate, the club entity won't be created and the user will be notified. (otherwise, the page shows a confirmation message)